### DISCUSSION TITLE
CSP array format

### DISCUSSION THREAD

**Discussion by incoming-th** [BePsvPT/secure-headers#81] [Q&A] [answered]
Hi,

Quick question, for the CSP array format should it be like this:

`
'csp' => [
    'default-src' => [
            'self',
            config('app.url')
    ],
`

or like this:

`
'csp' => [
    'default-src' => [
            'self' => true,
            'allow' => [
                     config('app.url')
             ]
    ],
`

Thanks

**Comment by bepsvpt** [comment, marked as answer]
`'csp' => [ 'default-src' => [ 'self' => true, 'allow' => [ config('app.url') ] ],` is the correct one.

In most cases, `'self' => true` will allow your website to load assets from the same domain.
